lib/jazzy/sourcekitten.rb

```diff
@@ -320,14 +320,18 @@ def self.deduplicate_declarations(declarations)
     def self.deduplication_key(decl)
       if decl.type.swift_extensible? || decl.type.swift_extension?
         [decl.usr, decl.name]
+      elsif decl.type.objc_class? || decl.type.objc_category?
+        name, = decl.objc_category_name || decl.name
+        [name, 'objc_class_and_categories']
       else
         [decl.usr, decl.name, decl.type.kind]
       end
     end
 
+    # rubocop:disable Metrics/MethodLength
```

Comment:
This method might benefit from a refactoring pass in the future. Right to just disable this for now and get it out the door, though!